source/Camera/InfoUtilOther/ZividBenchmark/ZividBenchmark.cpp

```diff
@@ -459,6 +465,68 @@ namespace
         printCapture2DResults(allDurations);
     }
 
+    Zivid::Frame assistedCapture(Zivid::Camera &camera)
+    {
+        const auto parameters = Zivid::CaptureAssistant::SuggestSettingsParameters{
+            Zivid::CaptureAssistant::SuggestSettingsParameters::AmbientLightFrequency::none,
+            Zivid::CaptureAssistant::SuggestSettingsParameters::MaxCaptureTime{ std::chrono::milliseconds{ 800 } }
+        };
+        const auto settings = Zivid::CaptureAssistant::suggestSettings(camera, parameters);
+        return camera.capture(settings);
+    }
+
+    template<typename DataType>
+    void CopyDataTime(Zivid::Frame frame, DataType copyName)
+    {
+        std::vector<Duration> copyDataDurations;
+
+        auto pointCloud = frame.pointCloud();
+        const auto beforeCopyData = HighResClock::now();        
+        const auto tempCopy = pointCloud.copyData<decltype(copyName)>();
+        const auto afterCopyData = HighResClock::now();
+
+        copyDataDurations.push_back(afterCopyData - beforeCopyData);
+        printCopydataResults(copyDataDurations);
+    }
+
+    void CopyDataTime_image(Zivid::Frame2D frame2D)
```

Comment:
call this `copyDataTime2D` maybe since we use `frame` and `frame2D`?

Reply:
Or even better, you can actually call it `copyDataTime` and since it has different input arguments it will work fine. This is ok thing to do in C++, to have two functions with the same name doing different things.